crates/holochain/src/conductor/space.rs

```diff
@@ -112,7 +112,7 @@ impl Spaces {
     }
 
     /// Get something from every space
-    pub fn get_from_spaces<R, F: FnMut(&Space) -> R>(&self, f: F) -> Vec<R> {
+    pub fn get_from_spaces<R, F: Fn(&Space) -> R>(&self, f: F) -> Vec<R> {
```

Comment:
why `Fn` now instead of `FnOnce`?

Reply:
:face_palm: - yep gottcha. This is my pet-peeve with "functional" programming, I feel like it sacrifices readability for conciseness... If there were a for-loop there, that would be obvious.

Well, in this case @maackle - You may want to put it back to `FnMut` because then the users have the option to put mutable state in their closure.

Reply:
I actually couldn't get it to accept a FnMut, so I'm leaving it Fn for now -- if you can figure it out feel free to change it.

Reply:
Revert this commit for reference: `90a59a6ec0d2442fa85d0b94b1cff78ceb1639c5`